the trait bound &[u8; 27]: Chunk<'_> is not satisfied

This code

```rust
use logos::Logos;

#[derive(Logos, Debug, Clone, PartialEq)]
pub enum TestLogos {
    #[regex(r"THIS IS A LONG REGEX STRING")]
    Variant,
    #[error]
    Error,
}

fn main() {}
```

produces this error

```
error[E0277]: the trait bound `&[u8; 26]: logos::source::Chunk<'_>` is not satisfied
 --> src/main.rs:3:10
  |
3 | #[derive(Logos, Debug, Clone, PartialEq)]
  |          ^^^^^ the trait `logos::source::Chunk<'_>` is not implemented for `&[u8; 26]`
  |
  = help: the following implementations were found:
            <&'source [u8; 10] as logos::source::Chunk<'source>>
            <&'source [u8; 11] as logos::source::Chunk<'source>>
            <&'source [u8; 12] as logos::source::Chunk<'source>>
            <&'source [u8; 13] as logos::source::Chunk<'source>>
          and 12 others
  = note: this error originates in a derive macro (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0277]: the trait bound `&[u8; 27]: logos::source::Chunk<'_>` is not satisfied
 --> src/main.rs:3:10
  |
3 | #[derive(Logos, Debug, Clone, PartialEq)]
  |          ^^^^^ the trait `logos::source::Chunk<'_>` is not implemented for `&[u8; 27]`
  |
  = help: the following implementations were found:
            <&'source [u8; 10] as logos::source::Chunk<'source>>
            <&'source [u8; 11] as logos::source::Chunk<'source>>
            <&'source [u8; 12] as logos::source::Chunk<'source>>
            <&'source [u8; 13] as logos::source::Chunk<'source>>
          and 12 others
  = note: this error originates in a derive macro (in Nightly builds, run with -Z macro-backtrace for more info)

error: aborting due to 2 previous errors
```


It seems like the `Chunk` trait is only implemented up to `[u8; 16]`, but how am I supposed to parse longer regular expressions?

--- a/logos/src/source.rs
+++ b/logos/src/source.rs
@@ -219,20 +219,11 @@ impl<'source> Chunk<'source> for u8 {
     }
 }
 
-macro_rules! impl_array {
-    ($($size:expr),*) => ($(
-        impl<'source> Chunk<'source> for &'source [u8; $size] {
-            const SIZE: usize = $size;
+impl<'source, const N: usize> Chunk<'source> for &'source [u8; N] {
+    const SIZE: usize = N;
 
-            #[inline]
-            unsafe fn from_ptr(ptr: *const u8) -> Self {
-                &*(ptr as *const [u8; $size])
-            }
-        }
-    )*);
+    #[inline]
+    unsafe fn from_ptr(ptr: *const u8) -> Self {
+        &*(ptr as *const [u8; N])
+    }
 }
-
-impl_array!(
-    1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 25, 26,
-    27, 28, 29, 30, 31, 32
-);

